app/helpers/workflows_helper.rb

```diff
@@ -56,7 +57,13 @@ def test_status_badge(status)
       label_class = 'label-default'
       label = t('test_status.not_available')
     end
-    content_tag(:span, "Tests: #{label}", class: "test-status label #{label_class}")
+    url = LifeMonitor::Rest::Client.status_page_url(resource)
+    link_to(url, class: 'lifemonitor-status btn btn-default', target: '_blank', rel: 'noopener',
+            'data-tooltip' => 'Click to view in LifeMonitor') do
+      image_tag('logos/life_monitor_logo_no_text.png', class: 'icon lifemonitor-logo') +
```

Comment:
it should be defined in the `image_file_dictionary.yml` - makes it much easier to catch missing images and change them